tools/read_netcdf/nc_info.xml

```diff
@@ -0,0 +1,32 @@
+<tool id="extract_nc_info" name="Netcdf Info" version="0.1.0">
+    <requirements>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        $__tool_directory__/NC_info.exe $input
```

Comment:
Compiled binaries are unlikely to work everywhere (old HPCs for example), please consider to use conda for this.